### PR TITLE
Split Official Resources

### PR DESCRIPTION
**Pull request by peaceiris**
In this Pull Request, I split the Official Resources of GitHub Actions and add new sections.

@gep13 
Sorry for skipping an issue opening. But I'm sure we need this change.

@imbsky
This pull request includes your commit on #243. After merging this branch into the master branch, you will join the contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,23 +43,84 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
...
+#### Workflow Tool Actions
+
+Tool actions for your workflow.
+
+- [actions/checkout](https://github.com/actions/checkout) - Setup your repository on your workflow.
+- [actions/upload-artifact](https://github.com/actions/upload-artifact) - Upload artifacts from your workflow.
+- [actions/download-artifact](https://github.com/actions/download-artifact) - Download artifacts from your build.
+- [actions/cache](https://github.com/actions/cache) - Cache dependencies and build outputs in GitHub Actions.
+- [actions/github-script](https://github.com/actions/github-script) - Write a script for GitHub API and the workflow contexts.
+- [`actions/github`](https://github.com/actions/github) - Wraps actions-toolkit into an Action for common GitHub automations.
```
Comment (peaceiris) [inline README.md]: Do you have another idea to skip this error?
https://github.com/sdras/awesome-actions/runs/286220732#step:4:8

Comment (imbsky) [reply README.md]: I'm not familiar with that tool, but isn't there any way to disable a specific rule for only one line?

**Comment by imbsky** [comment]
Well, I have no strong purpose to add cache action. Cause I thought the action of the actions team was listed in Collection of Actions section.

File: README.md (hunk 1 above)
Comment (peaceiris) [reply README.md]: Linter is [sindresorhus/awesome-lint](https://github.com/sindresorhus/awesome-lint). There is no option like a ignore option.

Comment (imbsky) [reply README.md]: Ignore option is good for many people, so it's a good idea to create a PR and improve that package. At least I'm not very much in favor of a change that breaks the unity of the current style.

Comment (peaceiris) [reply README.md]: By https://github.com/sindresorhus/awesome-lint/pull/78, we will be able to use the lint disable statement.

```md
<!--lint disable awesome-list-item-->
- [`actions/github`](https://github.com/actions/github) - Wraps actions-toolkit into an Action for common GitHub automations.
<!--lint enable awesome-list-item-->
```

**Comment by gep13** [comment]
@peaceiris thank you for taking the time to do this!  Looks great!

File: README.md (hunk 1 above)
Comment (peaceiris) [reply README.md]: Go to #246

**Pull request merged**